packages/ui/src/components/wui-loading-thumbnail/index.tsx

```diff
@@ -2,18 +2,27 @@ import { useEffect, useRef, type ReactNode } from 'react';
 import { Animated, Easing, View } from 'react-native';
 import Svg, { Rect } from 'react-native-svg';
 import { useTheme } from '../../hooks/useTheme';
+import { BorderRadius, Spacing, WalletImageSize } from '../../utils/ThemeUtil';
 import styles from './styles';
 
 const AnimatedRect = Animated.createAnimatedComponent(Rect);
 
 export interface LoadingThumbnailProps {
   children?: ReactNode;
   paused?: boolean;
+  borderRadius?: number;
 }
 
-export function LoadingThumbnail({ children, paused }: LoadingThumbnailProps) {
+export function LoadingThumbnail({
+  children,
```

Comment:
@svenvoskamp maybe this component refactor helps you for the one you have in web